services/enrichment/fields/budget/serverless.yml

```diff
@@ -0,0 +1,61 @@
+service: enrichment-fields-budget
+
+plugins:
+  - serverless-webpack
+
+custom:
+  esIndex: ${self:provider.stage}-projects
+  webpack:
+    webpackConfig: ./webpack.config.js
+    includeModules:
+      forceExclude:
+        - aws-sdk
+    packager: yarn
+  eubfrEnvironment: ${opt:eubfr_env, file(../../../../config.json):eubfr_env, env:EUBFR_ENV, 'dev'}
+  queueName: ${self:provider.stage}-enrichment-saver-backlog
+
+package:
+  individually: true
+
+provider:
+  name: aws
+  runtime: nodejs8.10
+  stage: ${opt:stage, file(../../../../config.json):stage, 'dev'}
+  region: ${opt:region, file(../../../../config.json):region, 'eu-central-1'}
+  deploymentBucket:
+    name: eubfr-${self:custom.eubfrEnvironment}-deploy
+  stackTags:
+    ENV: ${self:custom.eubfrEnvironment}
+  iamRoleStatements:
+    - Effect: "Allow"
+      Action: "sqs:*"
+      Resource: "arn:aws:sqs:${self:provider.region}:*:*"
+    - Effect: "Allow"
+      Action: "es:*"
+      Resource: "arn:aws:es:${self:provider.region}:*:domain/*"
+
+functions:
+  onEnrichRecordRequested:
+    handler: src/events/onEnrichRecordRequested.handler
+    name: ${self:provider.stage}-${self:service}-onEnrichRecordRequested
```

Comment:
Quite elegant naming in the field- services I think, this way a new service is basically the same but with a different self:service name :+1: